ui/imports/shared/views/chat/MessageView.qml

```diff
@@ -695,6 +698,8 @@ Loader {
                             return qsTr("Unknown message. Try fetching more messages")
                         return root.quotedMessageText
                     }
+                    album: root.quotedMessageAlbumMessageImages
+                    albumCount: root.quotedMessageAlbumImagesCount
```

Comment:
I couldn't understand why we need a separate `count` property? 
Wouldn't `...Image.length` work?

Reply:
It's used to show a loading indicator in case we haven't received all the images yet. Maybe in the case of replies we don't need the loading one though?

Reply:
That's right, when the image is gone missing or something..., if we have the image count, then we can have the loading style of images. I would recommend we keep this as this functionality is implemented already.
Coupling the image(s) and the count means that if we lose images then we can't have the loading images right?

Reply:
oh ok, didn't know that